### PR TITLE
Added authBody field to Technical struct

### PR DESCRIPTION
**Pull request by LtChae**
smartcitiesdata/smartcitiesdata#438

### CODE REVIEW

File: lib/smart_city/dataset/technical.ex [hunk 1]
```diff
@@ -30,6 +31,7 @@ defmodule SmartCity.Dataset.Technical do
   @derive Jason.Encoder
   defstruct allow_duplicates: true,
             authHeaders: %{},
+            authBody: %{},
```
Comment (mcsearchin) [inline lib/smart_city/dataset/technical.ex]: May want to update the moduledoc in `SmartCity.Dataset` to show this field is a part of the struct... not sure it's the right place for that doc, but that's where we've been doing it this far.

Comment (bbrewer-papa-x) [reply lib/smart_city/dataset/technical.ex]: True

File: mix.exs [hunk 2]
```diff
@@ -4,7 +4,7 @@ defmodule SmartCity.MixProject do
   def project do
     [
       app: :smart_city,
-      version: "3.14.1",
+      version: "3.15.0",
```
Comment (mcsearchin) [inline mix.exs]: Also, we may want to update the version on the `SmartCity.Data` struct itself (from 0.5 to 0.6?) Not sure we have normed on when to do that.  Just wanted to make sure we're thinking about it.

File: lib/smart_city/dataset/technical.ex (hunk 1 above)
Comment (LtChae) [reply lib/smart_city/dataset/technical.ex]: Ah, will do.

**Pull request merged**